tests/integration/test_neo4j_driver.py

```diff
@@ -45,3 +49,30 @@ def test_neo4j_uri(neo4j_uri, auth, target):
             pytest.skip(error.args[0])
         elif isinstance(error.__cause__, BoltHandshakeError):
             pytest.skip(error.args[0])
+
+
+def test_supports_multi_db(neo4j_uri, auth, target):
+    # python -m pytest tests/integration/test_neo4j_driver.py -s -v -k test_supports_multi_db
+    try:
+        driver = GraphDatabase.driver(neo4j_uri, auth=auth)
+        assert isinstance(driver, Neo4jDriver)
```

Comment:
Ok, I see why you assert in test above. But shouldn't you really just care that both URIs work regardless of their impl class?

Reply:
Depending on what uri it parsed it will return different driver objects with different capabilities. So there is the BoltDriver and Neo4jDriver. This could be merged into one DriverObject in the future.